compiler/aarch64/codegen/UnaryEvaluator.cpp

```diff
@@ -184,6 +184,29 @@ TR::Register *OMR::ARM64::TreeEvaluator::vnotEvaluator(TR::Node *node, TR::CodeG
    return inlineVectorUnaryOp(node, cg, notOp);
    }
 
+TR::Register*
+OMR::ARM64::TreeEvaluator::vsqrtEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL_WITH_NODE(node, node->getDataType().getVectorLength() == TR::VectorLength128,
+                           "Only 128-bit vectors are supported %s", node->getDataType().toString());
+   TR::InstOpCode::Mnemonic sqrtOp;
+
+   switch(node->getDataType().getVectorElementType())
+      {
+
```

Comment:
I don't think you need an empty line here.

Reply:
Removed the empty line.